### DISCUSSION TITLE
【知识库无法使用】按照流程正常安装，打开右上角知识库，回答显示。。。

### DISCUSSION THREAD

**Discussion by aden6966** [wenda-LLM/wenda#253] [Q&A]
按照流程正常安装，使用正常，可正常回答问题。
然后放入txt内容到txt文件夹，并跑了builds rtst bat文件，重新启动后打开网页右上角知识库按钮，所有的问题都无法回答，显示。。。。。
求助如何解决，需要修改哪些配置才能正常使用知识库。

**Comment by l15y** [comment]
更新至最新版并提供报错
